src/server/internalServer.cpp

```diff
@@ -658,7 +637,7 @@ std::vector<std::string>
 InternalServer::search_catalog(const RequestContext& request,
                                kiwix::OPDSDumper& opdsDumper)
 {
-    auto filter = kiwix::Filter().valid(true).local(true).remote(true);
+    auto filter = kiwix::Filter().valid(true).local(true);
```

Comment:
@mgautierfr kelson discussed this point earlier in #530 and we agreed that we should show all zim files and show a disabled download button(and a tooltip) for those zim files that can't be downloaded.

Reply:
On the kiwix-serve side yes. But OPDS stream is also used (and designed for) native client. You cannot change the behavior here as you may break them.

Reply:
@mgautierfr ok, what is broken concretly? What would be a solution?